locales-pending/premium.ftl

```diff
@@ -41,6 +41,11 @@ exposure-chart-legend-value-nr = { $nr }x
 
 # Here's What We Fixed Progress Card
 
+exposure-card-company-logo = Company logo
+exposure-card-company = Company domain
+exposure-card-exposure-type = Exposure types
+exposure-card-date-found = Date of the exposure
```

Comment:
@codemist These strings were missing in the `.ftl` files and I’ve seen a warning about it in the console. Are these strings correct or could you point me to the content?